### PR TITLE
Update event design to make organising partner clearer

### LINKED ISSUE #2426: Every event has a listed partner creator/owner
## Acceptance criteria

- [ ] On an event show page I can see contact info, its address, any linked partner and the organiser

## Design suggestion for event where whole event is shown

Screenshot 2024-04-17 at 17 35 57

### PR DESCRIPTION
**Pull request by r-ferrier**
Fixes #2426 

## Description

Adds another column to the event page, just to make it clear who the partner owner of an event is, ie who is the source of its calendar.

### CODE REVIEW

File: app/views/events/show.html.erb [hunk 1]
```diff
@@ -35,6 +35,12 @@
         %>
       </div>
     </div>
+    <div class="gi gi__1-3">
+      <h3 class="h4 udl">Event organiser</h3>
+      <div class="small">
+        <span><%= link_to @event.partner, @event.partner %></span>
```
Comment (ivan-kocienski-gfsc) [inline app/views/events/show.html.erb:41]: do events always have partners?

**Review by ivan-kocienski-gfsc — APPROVED** [review]
This sounds very useful, just one question (always with the edge cases)

File: app/views/events/show.html.erb (hunk 1 above)
Comment (r-ferrier) [reply app/views/events/show.html.erb:41]: yes!

Comment (r-ferrier) [reply app/views/events/show.html.erb:41]: can't have an an event without a calendar, can't have a calendar without a partner

Comment (ivan-kocienski-gfsc) [reply app/views/events/show.html.erb:41]: that makes sense

Comment (ivan-kocienski-gfsc) [reply app/views/events/show.html.erb:41]: that's a bit weird: the relation is optional but we have a validation for its presence?

Comment (r-ferrier) [reply app/views/events/show.html.erb:41]: 🤷‍♀️

**Pull request merged**